universal-login-relayer/lib/core/services/validators/IMessageValidator.ts

```diff
@@ -1,7 +1,7 @@
 import {SignedMessage} from '@universal-login/commons';
 
 export interface IMessageValidator {
-  validate: (message: SignedMessage) => Promise<void>;
+  validate: (message: SignedMessage) => Promise<void> | void;
```

Comment:
I would propose stick to Promise<void>. You can simply make `GasValidator.validate` async.